client/sections.js

```diff
@@ -251,6 +251,7 @@ const sections = [
 		module: 'my-sites/checkout',
 		secondary: true,
 		group: 'sites',
+		enableLoggedOut: true,
```

Comment:
Just making sure that this won't unexpectedly expose other pages to logged out users.

Reply:
I've verified that all /checkout routes redirect to log in, except for the quick start route. The condition for this is [here](https://github.com/Automattic/wp-calypso/pull/33749/files#diff-137f56ac9700496d02aebf4e8ad62b04R38). Also pinging team-calypso to make sure there are no gotchas.

Reply:
I heard back that there are no issues with this approach in C03DY3RJ5/p1560791734013400-slack.